gatsby-config.js

```diff
@@ -31,36 +32,36 @@ module.exports = {
     {
       resolve: 'gatsby-source-filesystem',
       options: {
-        name: 'meetings',
-        path: `${__dirname}/src/meetings`
+        name: 'meeting',
```

Comment:
i assume you have a reason behind the plural -> singular change. im curious about your thoughts on it

Reply:
I can remove this! More of a personal preference than anything meaningful. I think it makes more sense (to me) to query for a type of `meeting` instead of a type of `meetings`, too. But not a substantive enough reason to change!

Reply:
i dont think i have a strong preference here either, mostly because i havent gotten deep enough into things yet. i do tend to prefer plural in urls, for example, where an individual is made clear because of the presence of an id. not sure if that translates to this type of situation or not